### PR TITLE
Add the download information for WildFly 35 Beta

### PR DESCRIPTION
**Pull request by darranl**
Draft whilst I add the main announcement.

### CODE REVIEW

**Review by jamezp — APPROVED** [review]
LGTM

File: _posts/2024-12-12-WildFly35Beta-released.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+* Standard WildFly now link:https://docs.wildfly.org/wildfly-proposals/observability/WFLY-19846-promote-mp-tel-2-to-default.html[supports MicroProfile Telemetry 2.0, window=_blank]. This was previously supported in WildFly Preview.
+
+* Our MicroProfile Reactive Messaging subsystem has added OpenTelemetry tracing integration for link:https://docs.wildfly.org/wildfly-proposals/microprofile/WFLY-19835_microprofile_reactive_messaging_otel_with_kafka.html[Kafka connectors, window=_blank] and link:https://docs.wildfly.org/wildfly-proposals/microprofile/WFLY-19836_microprofile_reactive_messaging_otel_with_amqp.html[AMQP connectors, window=_blank].
+
+There's plenty of new things beyond MicroProfile area as well, though:
+
+* WildFly now includes link:https://docs.wildfly.org/wildfly-proposals/scripts/WFCORE-6935-system-daemon-scripts.html[supported `systemd` units] in the `bin/systemd` directory, replacing the old, unsupported `docs/contrib/scripts/systemd` files. The new units include support for a managed domain.
```
Comment (bstansberry) [inline _posts/2024-12-12-WildFly35Beta-released.adoc:31]: In the link adding ", window=_blank" would be good.

File: _posts/2024-12-12-WildFly35Beta-released.adoc [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+title:  "New WildFly 35 Beta release"
+date:   2024-12-12
+tags:   announcement release
+author: darranl
+description: WildFly 35 Beta is now available for download.
+---
+
+= WildFly 35 Beta is released!
+
+I'm pleased to announce that the new WildFly 35.0.0.Beta1 release is available for download at https://wildfly.org/downloads.
```
Comment (bstansberry) [inline _posts/2024-12-12-WildFly35Beta-released.adoc:12]: Adding ", window=_blank" would be good.

File: _posts/2024-12-12-WildFly35Beta-released.adoc [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+* We added link:https://docs.wildfly.org/wildfly-proposals/ee/WFLY-19776_Jakarta_Data_in_standard_WildFly.html[support for Jakarta Data 1.0, window=_blank] to standard WildFly. This was previously link:https://www.wildfly.org/news/2024/10/10/jakarta-data/[included in WildFly Preview 34, window=_blank]; now it is available in standard WildFly as well. This feature is provided at the link:https://docs.wildfly.org/34/Admin_Guide.html#Feature_stability_levels[`preview` stability level, window=_blank].
+
+* WildFly Preview has  a new link:https://docs.wildfly.org/wildfly-proposals/microprofile/WFLY-19954_Preview_Support_vertx_feature_pack.html[`vertx` subsystem and Galleon layer], intended to give user's greater control over the configuration of Vert.x instances running in the server. This feature is provided at the link:https://docs.wildfly.org/34/Admin_Guide.html#Feature_stability_levels[`preview` stability level, window=_blank].
```
Comment (bstansberry) [inline _posts/2024-12-12-WildFly35Beta-released.adoc:37]: In the link adding ", window=_blank" would be good.

File: _posts/2024-12-12-WildFly35Beta-released.adoc [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+* WildFly Glow has also received a new feature to https://github.com/wildfly/wildfly-proposals/pull/627[Introduce spaces to structure discovered Galleon feature-packs], this allows feature packs to be grouped into spaces such as an `experimental` space or a `default` space to reflect the stability of the feature pack and to allow users to select which spaces they want to use.
```
Comment (bstansberry) [inline _posts/2024-12-12-WildFly35Beta-released.adoc:39]: The link URL can now be https://docs.wildfly.org/wildfly-proposals/wf-galleon/WFLY-19855_glow_spaces.html.

s/Introduce/introduce

In the link adding ", window=_blank" would be good.

File: _posts/2024-12-12-WildFly35Beta-released.adoc [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+For further details please see the detailed https://github.com/wildfly/wildfly/releases/tag/35.0.0.Beta1[35.0.0.Beta1 Release Notes] in GitHub.
```
Comment (bstansberry) [inline _posts/2024-12-12-WildFly35Beta-released.adoc:41]: In the link adding ", window=_blank" would be good.

File: _posts/2024-12-12-WildFly35Beta-released.adoc [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+As we approach the end of 2024 this is the time of the year where we release the Beta release of the major version just before
+we wrap up for the end of the year and will follow up with the Final release once we return in the new year.  Please take this
+time as an opportunity to try out this release and provide us with any feedback.
+
+Finally I would like to thank everyone who has contributed to making this release happen, both via direct contributions to WildFly as well as the countless contributions to the projects that WildFly depends upon. We also had a couple of first time contributors this release so I would like to thank https://github.com/aldaris[Peter Major] for taking on one of our `good-first-issue` Jira issues to remove redundant code from our application client implementation and https://github.com/pedro-hos[Pedro Hos] for contributing updates to our testsuite to move tests to a more appropriate location.
```
Comment (bstansberry) [inline _posts/2024-12-12-WildFly35Beta-released.adoc:47]: In the links adding ", window=_blank" would be good.

File: _posts/2024-12-12-WildFly35Beta-released.adoc [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+* WildFly now link:https://docs.wildfly.org/wildfly-proposals/microprofile/WFLY-19591_MicroProfile_OpenAPI_4.0.html[supports MicroProfile OpenAPI 4.0, window=_blank].
+
+* Standard WildFly now link:https://docs.wildfly.org/wildfly-proposals/microprofile/WFLY-19866-promote-mp-rest-client-to-default.html[supports MicroProfile REST Client 4.0, window=_blank]. This was previously supported in WildFly Preview.
+
+* Standard WildFly now link:https://docs.wildfly.org/wildfly-proposals/observability/WFLY-19846-promote-mp-tel-2-to-default.html[supports MicroProfile Telemetry 2.0, window=_blank]. This was previously supported in WildFly Preview.
+
+* Our MicroProfile Reactive Messaging subsystem has added OpenTelemetry tracing integration for link:https://docs.wildfly.org/wildfly-proposals/microprofile/WFLY-19835_microprofile_reactive_messaging_otel_with_kafka.html[Kafka connectors, window=_blank] and link:https://docs.wildfly.org/wildfly-proposals/microprofile/WFLY-19836_microprofile_reactive_messaging_otel_with_amqp.html[AMQP connectors, window=_blank].
```
Comment (bstansberry) [inline _posts/2024-12-12-WildFly35Beta-released.adoc:27]: Nothing you need to deal with, but these two pages don't render properly (no title). I thought I fixed them last night but it seems not. I'll try again.

**Review by bstansberry — CHANGES_REQUESTED** [review]
The blog looks good, although I made a lot of comments. Most of which are for things you copied from my WIP yesterday! Sorry about that!

I verified the QS links in the new downloads page work. I can't for the server downloads as the release is not published yet on GitHub. But the urls look fine.

File: _posts/2024-12-12-WildFly35Beta-released.adoc (hunk 4 above)
Comment (jfdenise) [inline _posts/2024-12-12-WildFly35Beta-released.adoc:39]: I think we should say:
"...such as an `incubating` space to reflect..."

File: _posts/2024-12-12-WildFly35Beta-released.adoc (hunk 7 above)
Comment (bstansberry) [reply _posts/2024-12-12-WildFly35Beta-released.adoc:27]: I fixed the rendering on these, and posted on wildfly-dev about what the problem was.

**Pull request merged**